src/pages/modules/config.ts

```diff
@@ -0,0 +1,69 @@
+import { AxiosRequestConfig } from "axios";
+import { LocalStorage } from "../../app/common/LocalStorage";
+
+interface IApiConfig {
+  [desc: string]: AxiosRequestConfig;
+}
+
+const adminAPI = "https://edugator-admin.com"; // => base
+const token = LocalStorage.getToken();
+
+const noCache = { "Cache-Control": "no-cache" };
+const auth = { Authorization: "Bearer " + token };
```

Comment:
I set the token manually since the way `setAuthToken()` does it is wiped after a refresh. Admins coming here after closing the page won't have the header without this.

Reply:
There are other strategies that would rely on less code duplication like putting the authorization refresh (setAuthToken) call in a root or always rendered component like LayoutContainer or App or index https://stackoverflow.com/questions/54637071/axios-default-headers-cleared-after-page-refresh-in-react-js